core/modules/filter/tests/filter.test

```diff
@@ -1370,7 +1370,7 @@ class FilterUnitTestCase extends BackdropUnitTestCase {
     // - absolute, mail, partial
     // - characters/encoding, surrounding markup, security
 
-    // Create a e-mail that is too long.
+    // Create a email that is too long.
```

Comment:
"Create an email..."  rather than "Create a email..." (see other correct instances of "an email" in this PR)

Reply:
Good catch @argiepiano! I found one other instance that I fixed too.